packages/compass-sidebar/src/components/multiple-connections/connections-navigation.tsx

```diff
@@ -104,6 +105,42 @@ const noDeploymentStyles = css({
   gap: spacing[200],
 });
 
+function buildMonitoringUrl(atlasMetadata?: AtlasClusterMetadata) {
```

Comment:
Starting to think that maybe it's worth moving all those in a single place. Creating a new package seems like an overkill, maybe close to atlas service?